qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -275,18 +268,21 @@ def cancel(self) -> bool:
             raise IBMQJobApiError('Error cancelling job: %s' % error)
 
     def status(self) -> JobStatus:
-        """Query the API to update the status.
+        """Query the server for the latest job status.
 
         Note:
             This method is not designed to be invoked repeatedly in a loop for
-            an extended period of time. Doing so may cause an exception.
-            Use `wait_for_final_state()` if you want to wait for the job to finish.
+            an extended period of time. Doing so may cause the server to reject
+            your request.
+            Use :meth:`wait_for_final_state()` if you want to wait for the job to finish.
 
         Returns:
-            The status of the job, once updated.
+            The status of the job. If the job failed, you can use
+            :meth:`error_message()` to get more information.
```

Comment:
It might make more sense to place the second sentence under the `Note`'s section, rather than the return description?